<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-C582COR" failures="1" tests="5" name="ChromeTest" time="51.128" errors="0" timestamp="2022-09-23T22:58:47 IST">
  <testcase classname="TestPackge.TestNGClass2" name="createnewaccountdisplay" time="4.914"/>
  <testcase classname="TestPackge.TestNGClass2" name="loginpagedisplay" time="0.259"/>
  <testcase classname="TestPackge.TestNGclass" name="opencontacttohelpcenterpage" time="2.154"/>
  <testcase classname="TestPackge.TestNGclass" name="openmessgenerpage" time="2.838">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	url of openmessnger is not working expected [https://www.messenge.com] but found [https://www.messenger.com/]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
url of openmessnger is not working expected [https://www.messenge.com] but found [https://www.messenger.com/]
at TestPackge.TestNGclass.openmessgenerpage(TestNGclass.java:99)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- openmessgenerpage -->
  <testcase classname="TestPackge.TestNGClass2" name="featurepagedisplay" time="24.036"/>
</testsuite> <!-- ChromeTest -->
